include/boost/geometry/strategies/cartesian/side_non_robust.hpp

```diff
@@ -1,14 +1,14 @@
 // Boost.Geometry
 
-// Copyright (c) 2020, Oracle and/or its affiliates.
+// Copyright (c) 2020-2021, Oracle and/or its affiliates.
 
 // Contributed and/or modified by Vissarion Fysikopoulos, on behalf of Oracle
 
 // Licensed under the Boost Software License version 1.0.
 // http://www.boost.org/users/license.html
 
-#ifndef BOOST_GEOMETRY_STRATEGY_CARTESIAN_SIDE_NON_ROBUST_HPP
-#define BOOST_GEOMETRY_STRATEGY_CARTESIAN_SIDE_NON_ROBUST_HPP
+#ifndef BOOST_GEOMETRY_STRATEGIES_CARTESIAN_SIDE_NON_ROBUST_HPP
```

Comment:
Some time ago we agreed to move legacy strategies to `strategy/` directory. This is why you or Tinco created these strategies here.

This strategy should stay here and instead e.g. `side_by_triangle` should be moved.

We could do this for all strategies at once or move systematically the strategies that we modify. In case you prefered to do this now the original file should containg deprecation warning. See e.g. area strategies in both directories.